util/src/main/resources/tech/pegasys/teku/util/config/configs/mainnet.yaml

```diff
@@ -18,7 +18,7 @@ GENESIS_FORK_VERSION: 0x00000000
 GENESIS_DELAY: 604800
 
 # Genesis - Merge
-GENESIS_GAS_LIMIT: 30000000
+GENESIS_GAS_LIMIT: 10000000
```

Comment:
why `10000000` if spec says `30000000`? is it a Geth specific?

Reply:
Hm, yep, that was a value from Geth. Need to find out what's the right value

Reply:
Returned back the value. Good catch 👍